src/Symfony/Bridge/Twig/Tests/Extension/HttpKernelExtensionTest.php

```diff
@@ -46,7 +46,13 @@ public function testUnknownFragmentRenderer()
         ;
         $renderer = new FragmentHandler(array(), false, $context);
 
-        $this->setExpectedException('InvalidArgumentException', 'The "inline" renderer does not exist.');
+        if (method_exists($this, 'expectException')) {
+            $this->expectException('InvalidArgumentException');
```

Comment:
That could work in the cases where we dont need any parameters but this is then more in line with new PHPUnit prefered way. I would prefer to keep it as this but your call.